custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,184 @@
+            return PRESET_AUTO
+
+        if ventilation_configuration == OverkizCommandParam.STANDARD:
+            return PRESET_MANUAL
+
+        return None
+
+    @property
+    def preset_modes(self) -> Optional[List[str]]:
+        """Return a list of available preset modes."""
+        return PRESET_MODES
+
+    async def async_set_preset_mode(self, preset_mode: str) -> None:
+        """Set the preset mode of the fan."""
+        if preset_mode == PRESET_AUTO:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_VENTILATION_CONFIGURATION_MODE,
+                OverkizCommandParam.COMFORT,
+            )
+            await self._set_ventilation_mode(prog=OverkizCommandParam.OFF)
+
+        if preset_mode == PRESET_PROG:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_VENTILATION_CONFIGURATION_MODE,
+                OverkizCommandParam.STANDARD,
+            )
+            await self._set_ventilation_mode(prog=OverkizCommandParam.ON)
+
+        if preset_mode == PRESET_MANUAL:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_VENTILATION_CONFIGURATION_MODE,
+                OverkizCommandParam.STANDARD,
+            )
+            await self._set_ventilation_mode(prog=OverkizCommandParam.OFF)
+
+        await self.executor.async_execute_command(
+            OverkizCommand.REFRESH_VENTILATION_STATE,
+        )
+        await self.executor.async_execute_command(
+            OverkizCommand.REFRESH_VENTILATION_CONFIGURATION_MODE,
+        )
+
+    @property
+    def fan_mode(self) -> Optional[str]:
+        """Return the fan setting."""
+        ventilation_mode = self.executor.select_state(OverkizState.IO_VENTILATION_MODE)
+        cooling = ventilation_mode.get(OverkizCommandParam.COOLING)
+
+        if cooling == OverkizCommandParam.ON:
+            return FAN_BYPASS
+        else:
+            return OVERKIZ_TO_FAN_MODES[
+                self.executor.select_state(OverkizState.IO_AIR_DEMAND_MODE)
+            ]
+
+    @property
+    def fan_modes(self) -> Optional[List[str]]:
+        """Return the list of available fan modes."""
+        return [*FAN_MODES_TO_OVERKIZ]
+
+    async def async_set_fan_mode(self, fan_mode: str) -> None:
+        """Set new target fan mode."""
+        if fan_mode == FAN_BYPASS:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_AIR_DEMAND_MODE, OverkizCommandParam.AUTO
+            )
+        else:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_AIR_DEMAND_MODE, FAN_MODES_TO_OVERKIZ[fan_mode]
+            )
+
+        await self.executor.async_execute_command(
+            OverkizCommand.REFRESH_VENTILATION_STATE,
+        )
+        await self.executor.async_execute_command(
+            OverkizCommand.REFRESH_VENTILATION_CONFIGURATION_MODE,
+        )
```

Comment:
```suggestion
    async def async_set_fan_mode(self, fan_mode: str) -> None:
        """Set new target fan mode."""
        if fan_mode == FAN_BYPASS:
            await self.executor.async_execute_command(
                OverkizCommand.SET_AIR_DEMAND_MODE, OverkizCommandParam.AUTO
            )
            await self._set_ventilation_mode(cooling=OverkizCommandParam.ON)
        else:
            await self._set_ventilation_mode(cooling=OverkizCommandParam.OFF)
            await self.executor.async_execute_command(
                OverkizCommand.SET_AIR_DEMAND_MODE, FAN_MODES_TO_OVERKIZ[fan_mode]
            )

        await self.executor.async_execute_command(
            OverkizCommand.REFRESH_VENTILATION_STATE,
        )
```
For the FAN_BYPASS mode, we need to set cooling to ON, and we have to set it to OFF in other case.
Otherwise, the "REFRESH_VENTILATION_CONFIGURATION_MODE" is not needed when the FAN mode is changed.